spec/21-http_response_header_format.md

```diff
@@ -0,0 +1,184 @@
+# Trace Context HTTP Response Headers Format
+
+This section describes the binding of the distributed trace context to the `tracemeta` HTTP header.
+
+## Tracemeta Header
+
+The `tracemeta` HTTP response header field identifies a completed request in a tracing system. It has four fields:
+
+* `version`
+* `trace-id`
+* `parent-id`
+* `trace-flags`
+
+### Header Name
+
+Header name: `tracemeta`
```

Comment:
My preference is for something like `traceresponse` rather than `tracemeta`, as `tracemeta` is overly broad for this use case

Reply:
I like the name `traceresponse`. I was trying to think of something that implied describing an existing trace as opposed to defining a new one.